StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -54,11 +54,15 @@ final class PaymentSheetLoader {
                 // Overwrite the form specs that were already loaded from disk
                 switch intent {
                 case .paymentIntent(let elementsSession, _):
-                    _ = FormSpecProvider.shared.loadFrom(elementsSession.paymentMethodSpecs as Any)
+                    if !elementsSession.isBackupInstance {
+                        _ = FormSpecProvider.shared.loadFrom(elementsSession.paymentMethodSpecs as Any)
+                    }
```

Comment:
Is it possible for the backup session to ever have a valid form spec? Is it always guaranteed to be invalid?

Reply:
Based on my understanding of the code, if the call to elements/sessions fails, then we try hitting the official `v1/*_intents` api, which is guaranteed not to have specs.